mackerel-plugin-redash/README.md

```diff
@@ -6,9 +6,11 @@ redash stats custom metrics plugin for [mackerel-agent](https://github.com/macke
 ## Synopsis
 
 ```shell
-mackerel-plugin-redash [-metric-key-prefix=redash] [-timeout=5] [-uri=http://localhost/api/admin/queries/tasks?api_key=hoge]
+mackerel-plugin-redash [-metric-key-prefix=redash] [-timeout=5] [-uri=http://localhost/api/admin/queries/tasks?api_key=hoge] [-api-key=YOUR_REDASH_API_KEY]
```

Comment:
Same as https://github.com/mackerelio/mackerel-agent-plugins/pull/535/files#diff-3496c6f01c7e15e8d273911dfb989693R97, I'd like to request to remove `?api_key=hoge` from this example